configs/seL4Config.cmake

```
@@ -166,6 +166,18 @@ foreach(file ${result})
     include("${file}")
 endforeach()
 
+# Verify that, as a minimum any variables that are used
+# to find other build files are actually defined at this
+# point. This means at least: KernelArch KernelWordSize
+
+if ("${KernelArch}" STREQUAL "")
+    message(FATAL_ERROR "KernelArch must be set.")
```

might be nitpicking, but I'd change this error message to just say what is wrong here.
```suggestion
    message(FATAL_ERROR "Variable 'KernelArch' is not set.")
```